src/components/PhotoAlbum.css

```diff
@@ -0,0 +1,24 @@
+ul {
```

Comment:
```suggestion
.album {
    ul {
```
Recommend sticking to the modular css classes like above moving forward :)